scripts/spleeter/run.sh

```
@@ -1,41 +1,52 @@
+
+set -e
+
+model=${1:-2stems}
+
+case "$model" in
+  2stems) stems="vocals accompaniment" ;;
+  4stems) stems="vocals drums bass other" ;;
+  *) echo "usage: $0 [2stems|4stems]" >&2; exit 1 ;;
+esac
+
+if [ ! -f $model.tar.gz ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)